podcasts/Bookmarks/List/BookmarkListViewModel.swift

```diff
@@ -16,15 +20,15 @@ class BookmarkListViewModel: ObservableObject {
 
     init(bookmarkManager: BookmarkManager) {
         self.bookmarkManager = bookmarkManager
+        super.init()
 
         listenForAddedBookmarks()
     }
 
-    func reload() {
-        let bookmarks = episode.map { bookmarkManager.bookmarks(for: $0) } ?? []
+    override func reload() {
```

Comment:
Why this overrides a method?

It seems that `MultiSelectVM` and `ListVM` don't make any use of `reload` (and the method does nothing). Maybe we can just remove this?

Reply:
Haha, :spiderman-point: I removed it from the model. The main reason it overrode before was because when adding deletion I needed to also reflect changes to the multi select model. This has just been replaced with a didSet.